skt/kerneltree.py

```diff
@@ -329,6 +358,15 @@ def merge_patchwork_patch(self, uri):
             )
 
     def merge_patch_file(self, path):
+        """
+        Merge the particular local patch run.
```

Comment:
Same comments as with the `merge_patchwork_patch` apply here

Reply:
Yeah, I agree that the name of the method is slightly confusing. I think the merge prefix comes from the "merge" command, instead of git merge. And now we are back to the unclear naming which you mentioned before :)

Reply:
Ah I see.